include/RAJA/policy/atomic_builtin.hpp

```diff
@@ -149,6 +149,9 @@ struct BuiltinAtomicCAS<4> {
                            OPER const &oper,
                            ShortCircuit const &sc) const
   {
+#ifdef RAJA_COMPILER_MSVC
+#pragma warning( disable : 4244 )  // Force msvc to not emit conversion warning
+#endif
     unsigned oldval, newval, readback;
 
     oldval = RAJA::util::reinterp_A_as_B<T, unsigned>(*acc);
```

Comment:
@trws should the reinter_A_as_B be in camp; i.e., is it more generally useful?

Reply:
I'm not sure, I'll have to take a look at it, adding it to my list.